recipe/conda_build_config.yaml

```diff
@@ -8,3 +8,5 @@ numpy:
   - 1.16
 MACOSX_SDK_VERSION:            # [osx and x86_64]
   - 11.0                       # [osx and x86_64]
+is_python_min:
+  - no
```

Comment:
It looks like this is already on lines 5-6 above

https://github.com/conda-forge/python-feedstock/blob/57e96d5738f62e2b4c1cc164f598ddb4afb5b63e/recipe/conda_build_config.yaml#L5-L6

Given this, can we drop this?

```suggestion
```

Reply:
Good catch, thanks @jakirkham!

Reply:
Happy to help 🙂